mods/sgssb/moves.js

```diff
@@ -615,7 +615,7 @@ exports.BattleMovedex = {
 		id: "aquasubscribe",
 		name: "Aqua Subscribe",
 		priority: 1,
-		secondary: {
+		self: {
```

Comment:
Thats not what i meant....

I mean add ``target: "self",`` INSIDE the secondary object